src/ServiceControl.Audit/Infrastructure/SetupBootstrapper.cs

```diff
@@ -101,7 +101,7 @@ bool ValidateLicense(Settings.Settings settings)
                     return false;
                 }
 
-                if (LicenseManager.TryImportLicenseFromText(settings.LicenseFileText, out var importErrorMessage))
+                if (!LicenseManager.TryImportLicenseFromText(settings.LicenseFileText, out var importErrorMessage))
```

Comment:
Good catch, yeah they should be aligned

https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/SetupBootstrapper.cs#L79